airflow/contrib/operators/awsbatch_operator.py

```diff
@@ -81,8 +85,8 @@ class AWSBatchOperator(BaseOperator):
     template_fields = ('job_name', 'overrides',)
 
     @apply_defaults
-    def __init__(self, job_name, job_definition, job_queue, overrides, max_retries=4200,
-                 aws_conn_id=None, region_name=None, **kwargs):
+    def __init__(self, job_name, job_definition, job_queue, overrides, array_properties=None,
+                 max_retries=4200, aws_conn_id=None, region_name=None, **kwargs):
```

Comment:
If you want/need to change the order of arguments of a function you should mention that in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md), because it is a backwards-incompatible change. Note that backwards-incompatible changes also won't make it in 1.10.x. The next version where it is possible to include this in, would be 2.0.

Reply:
@feluelle 
Thanks for the comment.

Can I move the `array_properties` after `region_name` to make it backwards-compatible?

Thanks

Reply:
It's backward compatible, because init operator method must use a keyword arguments . It's a requirements defined by apply_defaults decorator.

Reply:
You're right @mik-laj - forget about that. 🤦‍♂ Sorry @penghou620 then I am fine with that.